test/mocha.opts

```diff
@@ -0,0 +1,5 @@
+--require ts-node/register
+--watch-extensions ts
+--recursive
+--reporter spec
```

Comment:
maybe tweak to `dot` (I believe) to minimize output so as not to fail on CI/CD services that fail for long logs